spec/mailers/placements/school_user_mailer_spec.rb

```diff
@@ -240,34 +240,69 @@
 
   describe "#placement_information_added_notification" do
     subject(:placement_information_added_notification) do
-      described_class.placement_information_added_notification(user, school, placements)
+      described_class.placement_information_added_notification(user, school, placements, academic_year)
     end
 
     let(:placements) { [create(:placement, school: school)] }
-    let(:school) { create(:placements_school, name: "School 1") }
+    let(:hosting_interest) { build(:hosting_interest, academic_year:, appetite: appetite) }
+    let(:school) { create(:placements_school, name: "School 1", hosting_interests: [hosting_interest]) }
```

Comment:
```suggestion
    let(:school) { build(:placements_school, name: "School 1", hosting_interests: [hosting_interest]) }
```

Could we utilise `build` here?